docs/quality-assurance/test-code-standards.md

```diff
@@ -31,11 +32,15 @@
 * Only use for e2e tests or explicit UI features at this level
 * Interactions with UIs should be abstracted into a separate service/facade, not in the test Steps (e.g., Page Object Model)
 
+## SpecFlow and Cucumber
+
+* Consider whether using these are required - will it add value, will a stakeholder be reading the tests, or can the extra technical layer be avoided?
+* If using, publish the tests to a website for easy access using Pickles (see our [Pickles Example Project](test-repositories.md))
+
 ## Gherkin
```

Comment:
We should remove the Gherkin tag and make this a single section. I think that the first two points (about "is it needed" and Pickles) should be emphasised in some manner as the rest of the section is irrelevant if the decision not to use it is taken

Reply:
Yep reads better, thanks.